src/app/components/Metadata/index.jsx

```diff
@@ -124,7 +123,7 @@ const Metadata = ({
       <meta name="og:image:alt" content={defaultImageAltText} />
       <meta name="og:locale" content={locale} />
       <meta name="og:site_name" content={brandName} />
-      <meta name="og:title" content={title} />
+      <meta name="og:title" content={pageTitle} />
```

Comment:
`og:title` and `twitter:title` should have same values as that for `<title>` - this was an oversight in the code that has now been addressed.